src/main/java/org/codehaus/mojo/flatten/FlattenMojo.java

```diff
@@ -356,6 +356,13 @@
     @Parameter( defaultValue = "${session}", readonly = true, required = true )
     private MavenSession session;
 
+    /**
+     * The core maven model readers/writers are discarding the comments of the pom.xml.
+     * By setting keepCommentsInPom to true the current comments are moved to the flattened pom.xml.
+     */
+    @Parameter( property = "flatten.dependency.keepComments", required = false )
```

Comment:
Could say what the default value is (false).

Reply:
I added defaultValue = "false" and provided the info in the javadoc 
( commit 522b1a46a264972f4ecbe15e35da58a095c6d8e6 )